include/nuttx/mutex.h

```diff
@@ -29,13 +29,14 @@
 #include <errno.h>
 #include <assert.h>
 #include <unistd.h>
-#include <nuttx/sched.h>
 #include <nuttx/semaphore.h>
```

Comment:
```suggestion
#include <nuttx/sched.h>
#include <nuttx/semaphore.h>
```

Reply:
i do it ,but have a complie error about 
![image](https://user-images.githubusercontent.com/75056955/170224772-b3e105a0-d590-441e-aa05-1163b46477cd.png)

Reply:
seems no issue now